xgcm/grid.py

```diff
@@ -2102,6 +2059,13 @@ def cumsum(self, da, axis, **kwargs):
     def _apply_vector_function(self, function, vector, **kwargs):
         # the keys, should be axis names
         assert len(vector) == 2
+        assert isinstance(vector, dict)
+
+        warnings.warn(
```

Comment:
Not sure why this is showing up as an addition? I thought we had those warnings already merged into main? Is the `grid_ufunc...` branch up to date with main?

Reply:
No it probably isn't up to date with main... I can fix that and then you merge `grid_ufunc_refactor_project` in to this branch?

Reply:
Sounds like a plan!